janusgraph-core/src/main/java/org/janusgraph/graphdb/query/vertex/MultiVertexCentricQueryBuilder.java

```diff
@@ -130,6 +130,7 @@ public JanusGraphMultiVertexQuery addAllVertices(Collection<? extends Vertex> ve
 
     @Override
     public Map<JanusGraphVertex, Iterable<JanusGraphEdge>> edges() {
+        check();
```

Comment:
I would like to check here the execution order. 
I’m not able to it before 03.01.2020

Reply:
Hello, thank you for reviewing my code. I don't think this check() method is elegant here.and  I changed the location of this method. I don't quite understand what you mean.